src/java.desktop/unix/classes/sun/print/CUPSPrinter.java

```diff
@@ -458,15 +480,36 @@ public static int getPort() {
         return cupsPort;
     }
 
+    /**
+     * Returns CUPS domain socket pathname.
+     */
+    private static String getDomainSocketPathname() {
+        return domainSocketPathname;
+    }
+
```

Comment:
You will need to suppress deprecation warnings here.

Reply:
Should I add `@SuppressWarnings("deprecation")` to the getDomainSocketPathname() method?
I see that CUPSPrinter class has `@SuppressWarnings("removal")` but its private method do not have any SuppressWarnings annotations.

Reply:
@SuppressWarnings("removal") is added for the isSandboxedApp() method.